{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[{"file_id":"1BijQF6s-T4-61QjsL2LXbHxbJVLiy5uy","timestamp":1722083932546}]},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"markdown","source":["## ***Load dataset***"],"metadata":{"id":"3_bxCSrkMcQx"}},{"cell_type":"code","source":["stat = ''\n","path = '/content/drive/MyDrive/Colab Notebooks/datos/pima-indians-diabetes.csv'\n","\n","try:\n","    # Load dataset\n","    df_diabetes = pd.read_csv(path, header=None, sep=',')\n","    stat = 'Loaded dataset'\n","\n","except FileNotFoundError:\n","    # Exception if file does not exist in path\n","    stat = 'Error: File not found. Check path, please'\n","\n","except Exception as e:\n","    # Other exceptions\n","    stat = 'Sorry, there was an error loading the dataset: ' + e\n","\n","# Status message\n","print(stat)"],"metadata":{"id":"95GEEQ3nORbG"},"execution_count":null,"outputs":[]}]}